test/tests/neutronics/output/tests

```diff
@@ -0,0 +1,51 @@
+[Tests]
+  [unrelaxed_tally_cell]
+    type = CSVDiff
+    input = cell.i
+    csvdiff = cell_out.csv
+    requirement = "The system shall allow for the output of the unrelaxed tally value for cell tallies."
```

Comment:
I think we already have tests covering the output of the raw tally values and standard deviations on both cell and mesh tallies. Can you please check, and if we do, remove these duplicative tests?

Reply:
I found tests for raw tally values and tally standard deviations for both; I'll get rid of the duplicates. 

Noting this as a future reference for myself and anyone else who plans on adding new derived tally objects: the unrelaxed tally output tests are in `tests/neutronics/relaxation/*`, the mesh tally standard deviation test is in `tests/neutronics/mesh_tally/fission_tally_std_dev.i`, and a test for cell tally standard deviations is in `tests/neutronics/tally_system/multi_diff.i` (using `cli_args` to change the output).